pkgs/applications/networking/tsung/default.nix

```diff
@@ -0,0 +1,26 @@
+{ fetchurl, stdenv, erlang }:
+
+stdenv.mkDerivation rec {
+  name = "tsung-${version}";
+  version = "1.7.0";
+  src = fetchurl {
+    url = "http://tsung.erlang-projects.org/dist/tsung-${version}.tar.gz";
+    sha256 = "6394445860ef34faedf8c46da95a3cb206bc17301145bc920151107ffa2ce52a";
+  };
+
+  buildInputs = [ erlang ];
+
+  meta = with stdenv.lib; {
+    homepage = "http://tsung.erlang-projects.org/";
+    description = "A high-performance benchmark framework for various protocols including HTTP, XMPP, LDAP, etc.";
+    longDescription = ''
+      Tsung is a distributed load testing tool. It is protocol-independent and
+      can currently be used to stress HTTP, WebDAV, SOAP, PostgreSQL, MySQL,
+      AMQP, MQTT, LDAP and Jabber/XMPP servers.
+    '';
+    license = licenses.gpl2;
+    maintainers = [ maintainers.uskudnik ];
+    priority = 1;
```

Comment:
What's the rationale behind this priority?

Reply:
Since this was my first new package I was basing my work on something similar, in this case [`jmeter`](https://github.com/NixOS/nixpkgs/blob/master/pkgs/applications/networking/jmeter/default.nix#L57) and per my understanding of [`meta.priority`](https://nixos.org/nix/manual/#description-1) the non-versioned package should have the highest priority which in this case would be base (latest) package. Though I originally also missed the sentence that default `priority` is 0. :blush: As such, I removed it.